app/models/item_order.rb

```diff
@@ -8,9 +8,5 @@ def subtotal
     price * quantity
   end
 
-  def have_enough?
-    item.inventory < quantity
-  end
-
```

Comment:
Method wasn't working, but used similar logic in the view and test passed

Reply:
@IamNorma 
That's really strange that it wouldn't be working... would love to talk through this later during work time / see your alternate solution! 

Side note, I would think this method should say `item.inventory <= quantity`.. no?

Reply:
Yes @gabichuelas you are correct. It should have been <=, was my mistake!